flux_sdk/time_and_attendance/capabilities/job_management/interface.py

```diff
@@ -15,3 +18,14 @@ def get_job_attributes(self) -> GetJobAttributesResponse:
         :return: The response containing the job attributes from the third-party system.
         """
 
+    @abstractmethod
+    def get_job_attributes_values(self) -> GetJobAttributesValueResponse:
```

Comment:
```suggestion
    def get_job_attribute_values(self) -> GetJobAttributeValuesResponse:
```
This might be a naming preference for myself but just try to keep it consistent with the `get_job_attributes` api as well.

Reply:
yeah i think this is a good change